LanguageExt.Core/HKT/HKT.Extensions.tt

```diff
@@ -142,6 +142,37 @@ namespace LanguageExt
 }
 <#+
 
+    public struct StringWrapper
```

Comment:
:memo: I didn't see a way to define an extension method for use in T4, so I went "sneaky" instead.